packages/react-magma-docs/src/pages/api/checkboxes.mdx

```diff
@@ -395,7 +394,7 @@ Placeholders are specific keywords surrounded by curly braces.
         status={status}
         labelText="Colors"
       />
-      <div style={{marginLeft: '30px'}}>
+      <div style={{marginLeft: '32px'}}>
```

Comment:
Should this come out of the spacing object as well?

Reply:
I've been less consistent about this in docs examples, but yes, I can definitely change this and will be more consistent going forward.